### PR TITLE
m4a covert art support

### PR DESCRIPTION
**Pull request by danimateo**
fixed bug where m4a cover art wouldn't show because tag is `-CoverArt` not `-Picture`

### CODE REVIEW

File: config/awesome/floppy/widget/mpd/mpd-music-updater.lua [hunk 1]
```diff
@@ -15,7 +15,7 @@ local apps = require('configuration.apps')
 
 local update_cover = function()
 	
-	local extract_script = [[
+	local extract_script = [=[
```
Comment (eromatiya) [inline config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: Why is there an equal there? Is it a typo?

Comment (danimateo) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: It's not a typo, that's how you can use `]]` for the bash if conditional inside the lua multistring. You can have any number of `=` as long as they are equal numbered.

Comment (eromatiya) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: Oh, I see. Thanks for the info! Never knew that before.

Comment (danimateo) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: You can check it [here](http://lua-users.org/wiki/StringsTutorial) at **Nesting quotes** (lua's weird ikr)

Comment (eromatiya) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: Thanks! Yeah lua's weird. Anyway, thanks for this. I'll just apply the patch to the other setups.

Comment (danimateo) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: Always glad to help :) Love my WM thanks to you, least I can do.

Comment (eromatiya) [reply config/awesome/floppy/widget/mpd/mpd-music-updater.lua:18]: Nice! Also, thanks for the nesting quotes. It's time to clean those bash scripts inside lua lol

**Pull request merged**